x-pack/solutions/security/plugins/security_solution/public/detections/components/alert_summary/kpis/kpis_section.tsx

```diff
@@ -16,21 +17,15 @@ import { AlertsProgressBarByHostNamePanel } from './alerts_progress_bar_by_host_
 
 export const KPIS_SECTION = 'alert-summary-kpis-section';
 
-export interface KPIsSectionProps {
-  /**
-   * DataView created for the alert summary page
-   */
-  dataView: DataView;
-}
-
 /**
  * Section rendering 3 charts in the alert summary page.
  * The component leverages existing chart components from the alerts page but is making a few tweaks:
  * - the SeverityLevelPanel and AlertsByRulePanel are used directly from the alerts page
  * - the UI differences on the AlertsProgressBarPanel were significant enough that a separate component was created
  */
-export const KPIsSection = memo(({ dataView }: KPIsSectionProps) => {
-  const signalIndexName = dataView.getIndexPattern();
+export const KPIsSection = memo(() => {
+  const spaceId = useSpaceId();
+  const signalIndexName = `${DEFAULT_ALERTS_INDEX}-${spaceId}`;
```

Comment:
What's the reasoning behind this change? I understand that the result is basically the same, but why not still retrieving the value from the `dataView`? Is that because in the latest code we're creating a dataView with `SourcererScope.detections`?

Also you might want to memoize the value here.

Reply:
If we make a change in the parent component, we'll have to remember and come make another change here. I kinda liked the fact that this component was driven by whatever was set in the parent component...

Reply:
I slighted organized the logic: alert summary has all the calls to data view, and it passes signal index to kpi section (it only needed signal index anyway). 

[This PR](https://github.com/elastic/kibana/pull/221338/files#diff-ce174f58194dc883b9bbf4749a59c42981890b24c4b1da5b7cda09514e9199dbR11) introduces the signal index slice in redux, so we could fetch it from there once it is merged. 

KPI charts needed index name for the query though, it doesn't rely on the data view

Reply:
That all makes sense, only pass down the minimum information needed, nice improvement!